tests/test-sparsek_kq_mask.cpp

```diff
@@ -0,0 +1,282 @@
+// tests/test-sparsek_kq_mask.cpp
```

Comment:
As SparseK is special case of attention, no need to create new test case cpp file.
1. Move it in test-backend-ops.cpp.
2. Follow the existed framework of test-backend-ops.cpp.
  like `test_cases.emplace_back(new test_flash_attn_ext( hsk, hsv, nh, {nr2, nr3}, kv, nb, mask, sinks, max_bias, logit_softcap, prec, type_KV))`
  It help to create more cases for OPs and extend to support GPU in the future.